pkg/kor/configmaps_test.go

```diff
@@ -95,6 +101,11 @@ func createTestConfigmaps(t *testing.T) *fake.Clientset {
 					ValueFrom: &corev1.EnvVarSource{ConfigMapKeyRef: &corev1.ConfigMapKeySelector{LocalObjectReference: corev1.LocalObjectReference{Name: configmap2.ObjectMeta.Name}}},
 				},
 			},
+			EnvFrom: []corev1.EnvFromSource{
```

Comment:
Nit: Can you make it used in a separate pod?

Reply:
Done